spec/requests/case_log_controller_spec.rb

```diff
@@ -240,6 +240,31 @@
           end
         end
       end
+
+      context "tenancy information pages" do
+        let(:headers) { { "Accept" => "text/html" } }
+        let!(:case_log_22_23_period) do
+          FactoryBot.create(
+            :case_log,
+            :about_completed,
+            startdate: Time.zone.local(2022, 10, 10),
+            owning_organisation: organisation,
+            managing_organisation: organisation,
+          )
+        end
+
+        it "has a joint tenancy page that can be visited if the case log is in the 2022-2023 period" do
```

Comment:
Same with this spec. Not sure it's adding much over the one that's there already? Also I think it would probably belong in `form_controller_spec` (form navigation) rather than `case_log_controller_spec` (case log data API)

Reply:
I think there's a slight difference where one is checking the difference in headers for the same question whereas this is checking the difference in presence of entire pages in a section. Happy to remove/move this though

Reply:
In both cases the behaviour actually being checked is that the correct form for that year is being routed to though right?

Reply:
Yep, that's a fair point. Happy to remove these in that case